### ISSUE TITLE
Draw annotations and Detections

### ISSUE THREAD

**Issue by VWJF** [coastal-science/cormorants-nesting-scripts#17]
# How to draw annotations (made by a person) or detections (made by a model)

A panorama can be very big, to present the panoramas to a user (for annotations) or to a model (for training or inference), it is broken into fixed-size tiles. It also happens to be that a panorama is constructed from smaller tiles.

* When a panorama is provided to the annotation tool, it is divided into fixed-size tiles (squares, except the tiles at the end of an axis).
* When a panorama is provided to the TensorFlow model (training or inference), it is also divided into fixed-size tiles.
* The annotation tile size and the model tile size may differ. 

Once selected, the `tile_size` and `anno_tile_size` hyperparameter must be reused consistently throughout the pipeline to correctly interpret coordinates. The default tile sizes are `anno_tile_size=3000` and `tile_size=1000`.

## Detections – Predictions made by the model

The TensorFlow detections are normalized to the full panorama's width and height. Meaning, they are $ (x,y) $ pairs where x and y are in the range $ [0,1] $. `x=1` means `x=tile_width`, and `y=1` means `y=tile_height`. To convert model coordinates to absolute panorama coordinates, they must be scaled by the tile_size, 

$$ pano_x = x * tile\_size_{width} $$
$$ pano_y = y * tile\_size_{height} $$

> TODO: In the csv, detection boxes are $ 0-100 $; however, the `create_detection_geom` docstring claims the boxes should be $ 0-1 $. We were expecting $0-1$.

## Annotations – Drawn by a person with an annotation tool (LOST or Label Studio)

The panoramas are annotated tile-by-tile (n.b. Label Studio annotations must be converted to LOST format to be usable by the model and pipeline). Within a tile, coordinates $ (x,y) $ are in the range $[0-1]$ where `x=1` means `x=tile_width`, and `y=1` means `y=tile_height`. Additionally, each tile is associated with its own tile-level offsets along 2D coordinates. To convert annotation coordinates to absolute panorama coordinates, the transformation is

$$ pano_x = x_0 + tile_x \times tile\_size_{width} $$ 

and correspondingly for y–coordinate and height. For example, coordinates (i,j) in the tile (2,3) are coordinates `i + 2 * tile_width,  j + 3 tile_height` in the scale of the absolute panorama $[0, pano_{width}], [0, pano_{height}]$.
